guides/source/configuring.md

```diff
@@ -66,7 +66,7 @@ These configuration methods are to be called on a `Rails::Railtie` object, such
 
 * `config.add_autoload_paths_to_load_path` says whether autoload paths have to be added to `$LOAD_PATH`. This flag is `true` by default, but it is recommended to be set to `false` in `:zeitwerk` mode early, in `config/application.rb`. Zeitwerk uses absolute paths internally, and applications running in `:zeitwerk` mode do not need `require_dependency`, so models, controllers, jobs, etc. do not need to be in `$LOAD_PATH`. Setting this to `false` saves Ruby from checking these directories when resolving `require` calls with relative paths, and saves Bootsnap work and RAM, since it does not need to build an index for them.
 
-* `config.cache_classes` controls whether or not application classes and modules should be reloaded on each request. Defaults to `false` in development mode, and `true` in test and production modes.
+* `config.cache_classes` controls whether or not application classes and modules should be reloaded on each request. Defaults to `false` in development and test modes, and `true` in production mode. If you installed Spring in test mode, Defaults to `true` in test mode.
```

Comment:
The latter sentence is probably "not install". So how about like the following?
"If you didn't install Spring gem, the default for test mode is changed to `true`."

Reply:
@y-yagi 

Thank you for your suggestions 🙇 
I Fixed description.

Reply:
Let me nitpick: the default value itself is not changed, that assumes there is something initial that gets modified. The default value _depends_ on whether you install spring or not.

Reply:
@fxn 
CC: @y-yagi 

Thank you comment! That makes sense.
What do you think about the following sentences?

> If you install the Spring gem then default to `false` else `true` in test mode.

Reply:
Revising wording is hard in a PR, eh? :). Better, but I'd like to propose a simplification:

> In `test` mode, the default is `false` if Spring is installed, `true` otherwise.

Reply:
@fxn 
CC: @y-yagi 

Thank you for your suggestions 🙇I fixed sentences.